### PR TITLE
roachtest: set lower min range_max_bytes in multitenant_distsql test

### LINKED ISSUE #99626: roachtest: multitenant/distsql/instances=20/bundle=off/timeout=1 failed
roachtest.multitenant/distsql/instances=20/bundle=off/timeout=1 [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9262923?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9262923?buildTab=artifacts#/multitenant/distsql/instances=20/bundle=off/timeout=1) on master @ [2bd2c806ab3044569b09e0a205b5bc0452ad4e2b](https://github.com/cockroachdb/cockroach/commits/2bd2c806ab3044569b09e0a205b5bc0452ad4e2b):

```
test artifacts and logs in: /artifacts/multitenant/distsql/instances=20/bundle=off/timeout=1/run_1
(assertions.go:262).Fail: 
	Error Trace:	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/multitenant_distsql.go:122
	            				github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/multitenant_distsql.go:44
	            				main/pkg/cmd/roachtest/test_runner.go:993
	            				GOROOT/src/runtime/asm_amd64.s:1594
	Error:      	Received unexpected error:
	            	pq: could not validate zone config: RangeMaxBytes 100000 less than minimum allowed 67108864
	Test:       	multitenant/distsql/instances=20/bundle=off/timeout=1
(require.go:1264).NoError: FailNow called
(cluster.go:1977).Run: cluster.RunE: context canceled
(cluster.go:1977).Run: cluster.RunE: context canceled
(cluster.go:1977).Run: cluster.RunE: context canceled
(cluster.go:1977).Run: cluster.RunE: context canceled
(cluster.go:1977).Run: cluster.RunE: context canceled
(cluster.go:1977).Run: cluster.RunE: context canceled
(cluster.go:1977).Run: cluster.RunE: context canceled
(cluster.go:1977).Run: cluster.RunE: context canceled
(cluster.go:1977).Run: cluster.RunE: context canceled
(cluster.go:1977).Run: cluster.RunE: context canceled
(cluster.go:1977).Run: cluster.RunE: context canceled
(cluster.go:1977).Run: cluster.RunE: context canceled
(cluster.go:1977).Run: cluster.RunE: context canceled
(cluster.go:1977).Run: cluster.RunE: context canceled
(cluster.go:1977).Run: cluster.RunE: context canceled
(cluster.go:1977).Run: cluster.RunE: context canceled
(cluster.go:1977).Run: cluster.RunE: context canceled
(cluster.go:1977).Run: cluster.RunE: context canceled
(cluster.go:1977).Run: cluster.RunE: context canceled
(cluster.go:1977).Run: cluster.RunE: context canceled
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>

See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*multitenant/distsql/instances=20/bundle=off/timeout=1.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

Jira issue: CRDB-25988

### PR DESCRIPTION
**Pull request by rharding6373**
The multitenant_distsql roachtest relies on a smaller range size than the new default min of range_max_bytes (64MiB) due to performance and resource limitations. We set the COCKROACH_MIN_RANGE_MAX_BYTES to allow the test to use the smaller range.

Fixes: #99626
Epic: None

Release note: None

### CODE REVIEW

**Comment by mgartner** [comment]
@aliher1911 Does this seem sane to you? I think we need to do that same thing you did in #99636 to fix #99626.

**Review by rharding6373 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/99759)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @aliher1911, @renatolabs, and @srosenberg)

___
*[`pkg/cmd/roachtest/tests/multitenant_distsql.go` line 126 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/99759#-NRdnr5h1D4IsoYazdiZ:-NRdnr5h1D4IsoYazdi_:b-75f4g0) ([raw file](https://github.com/cockroachdb/cockroach/blob/045e3930e1dcf38c3b929e7240e83352ac026100/pkg/cmd/roachtest/tests/multitenant_distsql.go#L126)):*
> ```Go
> 	// and test takes too long and too much resources with default range sizes
> 	// so make them much smaller.
> 	_, err = inst1Conn.Exec(`ALTER TABLE t CONFIGURE ZONE USING range_min_bytes = 1000,range_max_bytes = 100000`)
> ```

Just want to note that this is where we set the range_max_bytes in this test.

**Comment by aliher1911** [comment]
@mgartner It worked when run roachtests manually, but I'm yet to see it merge and pass nightly. If you want to be on a safe side, we can wait for my PR to merge and run overnight before merging. But I don't see why it could fail.

**Comment by rharding6373** [comment]
This passes the zone config checks when I test with roachtest manually, as well.

**Comment by mgartner** [comment]
I'd vote for merging this now. In the best case, we get less noise tomorrow. In the worst case, we have to find a new fix. At least we'll have more information.

**Review by renatolabs — APPROVED** [review]

**Comment by rharding6373** [comment]
TFTR!

bors r+

**Comment by rharding6373** [comment]
Will try again after flaky test fixes are in.

bors r-

**Comment by renatolabs** [comment]
Sadly this doesn't seem to fix this set of roachtests. The statement to change `range_max_bytes` is issued against the tenant connection, and that tenant is still created without the env var, so it will still use the new range size defaults.

(The fact that we have separate code in `multitenant_utils.go` to start tenants replicating a lot of the logic of `c.Start` is also unfortunate, but I digress.)

**Pull request merged**
